src/NavBar.ts

```diff
@@ -64,35 +69,6 @@ export default class NavBar {
 
             return point
         })
-
-        // support legacy URLs without coordinates (not initialized) and only text, see #199
-        if (points.some(p => !p.isInitialized && p.queryText.length > 0)) {
-            if (!profile.name) profile = { name: 'car' }
```

Comment:
This hard-coded 'car' profile was only used when there were point parameters with a query text and is removed now.